tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -44,6 +44,12 @@ sub get_script_run {
     my $guest_list = get_required_var("GUEST_LIST");
 
     $pre_test_cmd = "$pre_test_cmd -p $product_upgrade -r $product_upgrade_repo -g \"$guest_list\"";
+    my $do_registration = check_var('GUEST_SCC_REGISTER', 'installation') ? "true" : "false";
+    my $registration_server = get_var('SCC_URL', 'https://scc.suse.com');
```

Comment:
SCC_URL is always 'http://all-45.1.proxy.scc.suse.de/', is it that you want? or you are going to override it in your indevidual test?

Reply:
I do not think so that. There is not any code change about SCC_URL. Refer to http://10.67.129.59/tests/398#settings and http://10.67.129.59/tests/359#settings, this SCC_URL setting from my trigger command as below: 
Such as:
/usr/share/openqa/script/client --host http://10.67.129.59 isos post DISTRI=sle VERSION=15-SP6 FLAVOR=Online ARCH=s390x BUILD=45.1 _NO_OBSOLETE=1 REPO_0=SLE-15-SP6-Online-s390x-Build45.1-Media1 TEST=gi-guest_developing-on-host_developing-kvm
OR
/usr/share/openqa/script/client --host http://10.67.129.59 isos post DISTRI=sle VERSION=15-SP6 FLAVOR=Online ARCH=s390x BUILD=44.1 _NO_OBSOLETE=1 REPO_0=SLE-15-SP6-Online-s390x-Build44.1-Media1 TEST=gi-guest_developing-on-host_developing-kvm

Reply:
If your guests are sle15sp6, proxy scc is the right server; if your guest are sle12sp5/15sp5, you need scc.suse.com. I asked to just let you be aware whether proxy scc is what you would like to use at here.

Reply:
Super, gotcha. Make sense now. Thanks so much for your suggestion.

Reply:
@Julie-CAO  How do you suggest Leon to achieve that?

Reply:
talked with Leon offline, he said the guest is sle15sp6, so proxy scc is the exact what he wants.